gemini/getting-started/intro_gemini_pro_python.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<p>It'd be better to make is clear that this notebook is for Gemini pro only.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks, clarified this notebook focus on using Gemini API with Gemini Pro model. I remained brief introduction for two models to keep big picture for the Vertex AI Gemini API capabilities.</p>